src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,10 +22,23 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
```

Comment:
First, we use exactly the same naming convention for static or global variables as any other - that is camelCase.   CapCamelCase is reserved for ClassNames.  (I think this is unfortunate - most other shops use ALL_CAPS for constants - but it is what it is).

I think using `std::pair<std::string, std::string>` is a poor choice to represent the id and "friendly name" - which I think we call "human-readable name".

It's far too easy write `am.first` instead of `am.second` somewhere down the line and a reviewer might well miss it.  We have seen exactly such problems (using tuples, actually) in rippled before.

With C++11 and uniform initialization, any old struct is just as good as a `std::pair`.  Consider something like:

    struct AmendmentName 
    {
        std::string id;
        std::string humanName;
    };